workflows/Check_Host_Block_Status_with_Palo_Alto_Firewall_from_Microsoft_Teams/workflow.spec.yaml

```
@@ -0,0 +1,26 @@
+extension: workflow
+products: ["insightconnect"]
+name: Check_Host_Block_Status_with_Palo_Alto_Firewall_from_Microsoft_Teams
+title: "Check Host Block Status with Palo Alto Firewall from Microsoft Teams"
+description: This workflow accepts a Microsoft Teams command, checks if an IP address is blocked, and posts if the IP address is blocked.
```

```suggestion
description: Check if an IP address is blocked via Microsoft Teams.
```
I think this will read better in the hub